src/sentry/stacktraces/processing.py

```diff
@@ -257,17 +268,14 @@ def normalize_stacktraces_for_grouping(data: Any, grouping_config: Any = None) -
     """
 
     stacktrace_frames = []
-    stacktrace_exceptions = []
+    stacktrace_container = []
 
     with sentry_sdk.start_span(op=op, description="find_stacktraces_in_data"):
+        # XXX: Why do we want include the raw stacktrace?
         for stacktrace_info in find_stacktraces_in_data(data, include_raw=True):
-            # XXX: We already do this within find_stacktraces_in_data
-            frames = get_path(stacktrace_info.stacktrace, "frames", filter=True, default=())
-            if frames:
-                stacktrace_frames.append(frames)
-                stacktrace_exceptions.append(
-                    stacktrace_info.container if stacktrace_info.is_exception else None
-                )
```

Comment:
I think this has to do with https://github.com/getsentry/sentry/pull/15795#discussion_r349916876 as well, so I'd be wary of changing it unless you've investigated how it would affect the problems that PR was trying to solve.